### PR TITLE
refactor: focus state

### PR DESCRIPTION
**Pull request by leMaur**
## Summary
https://app.clickup.com/t/f92qxg

## Checklist

-   [x] I checked my UI changes against the design and there are no notable differences
-   [x] I checked my UI changes for any responsiveness issues
-   [x] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I provided a screenshot of my changes to the component _(if applicable)_
-   [ ] I regenerated the `icons.html` file and checked if my newly added icon is shown correctly _(if necessary)_
-   [ ] I added an explanation on how to use the component to the readme _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [x] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

**Comment by ItsANameToo** [comment]
@leMaur conflict

File: resources/views/inputs/input-with-icon.blade.php [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
             @if ($slot ?? false)
                 <div>
-                    <div class="flex {{ $slotClass ?? 'h-full bg-white' }}">
+                    <div class="flex {{ $slotClass ?? 'h-full' }}">
```
Comment (ItsANameToo) [inline resources/views/inputs/input-with-icon.blade.php:32]: when you remove this, it gives this issue in the footer
image

Comment (ItsANameToo) [reply resources/views/inputs/input-with-icon.blade.php:32]: entry is rounded and the icon has no background aspect anymore to be part of the "input"

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

File: resources/views/inputs/input-with-icon.blade.php (hunk 1 above)
Comment (leMaur) [reply resources/views/inputs/input-with-icon.blade.php:32]: @ItsANameToo the `bg-white` is now moved to msq

**Pull request merged**